crates/sargon/src/profile/v100/entity/persona/persona.rs

```diff
@@ -383,6 +383,7 @@ impl Ord for Persona {
                 .derivation_path()
                 .last_component()
                 .cmp(r.transaction_signing.derivation_path().last_component()),
+            _ => Ordering::Equal, // TODO check that
```

Comment:
TODO:
What the order should be when one of the personas is securified?

Reply:
see https://github.com/radixdlt/sargon/pull/235/files/164d0ef5159853b6187cb6c3705990fe769085a5#r1796760702